parachain/pallets/inbound-queue/src/mock.rs

```diff
@@ -130,7 +130,7 @@ impl Verifier for MockVerifier {
 const GATEWAY_ADDRESS: [u8; 20] = hex!["eda338e4dc46038493b885327842fd3e301cab39"];
 
 parameter_types! {
-	pub const EthereumNetwork: xcm::v3::NetworkId = xcm::v3::NetworkId::Ethereum { chain_id: 15 };
+	pub const EthereumNetwork: xcm::v3::NetworkId = xcm::v3::NetworkId::Ethereum { chain_id: 5 };
```

Comment:
Is it ok to hardcode the chain ID in tests? Do we want to use the constant here? I am not sure about adding the `rococo-common` crate here as a dependency just to use the chain ID.